ecommerce/utils.py

```diff
@@ -55,3 +57,24 @@ def get_order_id_by_reference_number(*, reference_number, prefix):
         raise ParseException("Unable to parse order number")
 
     return order_id
+
+
+def send_support_email(subject, message):
```

Comment:
I was originally going to add a function to `mitxpro/utils.py` and saw this function in there. IMO that was not a good place for this, so I went ahead and moved it here